system/applications/modules/codium/default.nix

```diff
@@ -44,18 +44,18 @@ in
       in
       {
         ${username} = mkIf (cfg.applications.codium.enable) {
-          # Enable wayland support for codium
+          # Codium profile used as a multi-purpose text editor
           xdg.desktopEntries.codium = {
```

Comment:
This desktop entry should be deleted entirely. We only need to specify an entry for IDE.

Reply:
I kept it because I wasn't sure if the name would be correct, but ai guess we don't really care anyways, will remove.